src/installers.ts

```diff
@@ -849,13 +849,141 @@ export const installJsonMod: VortexWrappedInstallFunc = (
 
   return Promise.resolve({ instructions });
 };
+
+// INI (includes Reshade?)
+export const testForIniMod: VortexWrappedTestSupportedFunc = (
+  api: VortexAPI,
+  log: VortexLogFunc,
+  files: string[],
+  _gameId: string,
+): Promise<VortexTestResult> => {
+  // Make sure we're able to support this mod.
+  const correctGame = _gameId === GAME_ID;
+  log("info", "Checking for INI files: ", _gameId);
+  if (!correctGame) {
+    // no mods?
+    return Promise.resolve({
+      supported: false,
+      requiredFiles: [],
+    });
+  }
+  const filtered = files.filter(
+    (file: string) => path.extname(file).toLowerCase() === INI_MOD_EXT,
+  );
+
+  if (filtered.length === 0) {
+    log("info", "No INI files.");
+    return Promise.resolve({
+      supported: false,
+      requiredFiles: [],
+    });
+  }
+
+  if (
+    files.some(
+      (file: string) =>
+        path.basename(file).includes(CET_MOD_CANONICAL_INIT_FILE) ||
+        path.extname(file) === REDS_MOD_CANONICAL_EXTENSION,
+    )
+  ) {
+    log("info", "INI file detected within a CET or Redscript mod, aborting");
```

Comment:
CET should already be handled at this point — I think we can and should rely on the priority

Reply:
So, pull this block?  I think Elie wanted this kind of test in the JSON handler just in case somebody packaged a CET mod wrong

Reply:
Hm, it doesn't hurt to leave it, but if we have a CET mod that escapes the CET installer, we should try to capture it in a test if it can't be fixed in CET.

Reply:
Either path is fine... or well, should def have a test that covers it. I will leave it up to y'all to keep the block or not.